src/main.rs

```diff
@@ -193,20 +194,24 @@ async fn main() -> std::io::Result<()> {
             Some(Arc::new(
                 CelestiaConnection::new(
                     &celestia_conf.connection_string,
-                    None,
+                    Some("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.d1x8PXcrFxyil3gSSCmPqHDJa5tsQV_iTAhRTnS5Mos"),
                     &celestia_conf.namespace_id,
                 )
                 .await,
             ) as Arc<dyn DataAvailabilityLayer + 'static>)
         }
+        DALayerOption::InMemory => Some(Arc::new(
+            InMemoryDataAvailabilityLayer::new(),
+        ) as Arc<dyn DataAvailabilityLayer + 'static>),
         DALayerOption::None => None,
     };
 
     let node: Arc<dyn NodeType> = match args.command {
         // LightClients need a DA layer, so we can unwrap here
         Commands::LightClient {} => Arc::new(LightClient::new(da.unwrap())),
         Commands::Sequencer {} => Arc::new(Sequencer::new(
-            Arc::new(RedisConnections::new()),
+            // convert error to std::io::Error...is there a better solution?
```

Comment:
```suggestion
            // TODO: convert error to std::io::Error...is there a better solution?
```

Adding TODOs before comments will let us search for TODO's easier in future

Reply:
true, i think this will be solved, when we have our own error enum after the next step